docs/_functions/responding.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Responding to interactivity
+lang: en
+slug: responding-to-interactivity
+order: 4
+---
+
+<div class="section-content">
+
+Interactive elements provided to the user from within the `function()` method’s callback are associated with that unique `function_executed` event. This association allows for the completion of functions at a later time, like once the user has clicked a button.
```

Comment:
Should we provide some context on _how_ an interactive element is provided to the user? E.g. "via a message or by opening a modal" or some such?